### DISCUSSION TITLE
password

### DISCUSSION THREAD

**Discussion by SkveeRLS** [wg-easy/wg-easy#1346] [Q&A] [answered]
where is the password stored, or how do I change it?

**Comment by tetuaoro** [comment, marked as answer]
Hi,

<s>In `WG_PATH/wg.conf` file. The default path is `/etc/wireguard/wg.conf`.</s>

It is not store in a file. There is an env to configure : `PASSWORD_HASH`.

**Reply by OJambrina** [reply]
hello, i would like to know, because i have homeassistant with the addon installed, how would i use the password hash
Error: DO NOT USE PASSWORD ENVIRONMENT VARIABLE. USE PASSWORD_HASH INSTEAD.

See https://github.com/wg-easy/wg-easy/blob/master/How_to_generate_an_bcrypt_hash.md
But in that guide I dont know how i could make that

Regards
